apps/src/templates/sectionProgress/standards/StandardsProgressTable.jsx

```diff
@@ -0,0 +1,187 @@
+  };
+
+  lessonsCompletedColumnFormatter = (lesson, {rowData, columnIndex}) => {
+    return <div id={rowData.id}>{rowData.numCompleted}</div>;
+  };
+
+  standardDescriptionColumnFormatter = (standard, {rowData, rowIndex}) => {
+    return (
+      <StandardDescriptionCell
+        id={rowData.id}
+        description={rowData.description}
+        lessonsForStandardStatus={rowData.lessonsForStandardStatus}
+      />
+    );
+  };
+
+  getColumns = () => {
+    let dataColumns = [
+      {
+        property: 'category',
+        header: {
+          label: i18n.standardConcept(),
+          props: {
+            style: {
+              ...tableLayoutStyles.headerCell,
+              ...styles.mainColumnHeader
+            }
+          }
+        },
+        cell: {
+          formatters: [this.standardCategoryCellFormatter],
+          props: {
+            style: {
+              ...tableLayoutStyles.cell,
+              ...styles.mainColumnCell
+            }
+          }
+        }
+      },
+      {
+        property: 'number',
+        header: {
+          label: i18n.standardIdentifier(),
+          props: {
+            style: {
+              ...tableLayoutStyles.headerCell,
+              ...styles.mainColumnHeader
+            }
+          }
+        },
+        cell: {
+          formatters: [this.standardNumberCellFormatter],
+          props: {
+            style: {
+              ...tableLayoutStyles.cell,
+              ...styles.mainColumnCell
+            }
+          }
+        }
+      },
+      {
+        property: 'description',
+        header: {
+          label: i18n.description(),
+          props: {
+            style: {
+              ...tableLayoutStyles.headerCell,
+              ...styles.descriptionColumnHeader
+            }
+          }
+        },
+        cell: {
+          formatters: [this.standardDescriptionColumnFormatter],
+          props: {
+            style: {
+              ...tableLayoutStyles.cell,
+              ...styles.descriptionCell
+            }
+          }
+        }
+      },
+      {
+        property: 'numCompleted',
+        header: {
+          label: i18n.lessonsCompleted(),
+          props: {
+            style: {
+              ...tableLayoutStyles.headerCell,
+              ...styles.mainColumnHeader
+            }
+          }
+        },
+        cell: {
+          formatters: [this.lessonsCompletedColumnFormatter],
+          props: {
+            style: {
+              ...tableLayoutStyles.cell,
+              ...styles.mainColumnCell
+            }
+          }
+        }
+      }
+    ];
+    return dataColumns;
+  };
+
+  getNumCompleted = (standard, index) => {
+    let count = 0;
+    if (this.props.lessonsCompletedByStandard[index + 1]) {
+      this.props.lessonsCompletedByStandard[index + 1].lessons.forEach(
+        lesson => {
+          if (lesson.completed) {
+            count++;
+          }
+        }
+      );
+    }
+    return count;
+  };
```

Comment:
I wonder if number of lessons completed will be something we calculate ahead of time and pass in rather than recalculating each time the table renders? Might have to see how the data shapes play out.

Reply:
I'm guessing this might move out to redux once we have a redux store for this